src/ConcreteOperations/issubset.jl

```diff
@@ -526,6 +526,46 @@ function ⊆(x::Interval, y::Interval, witness::Bool=false)
     return x.dat ⊆ y.dat
 end
 
+function ⊆(x::Interval, U::UnionSet, witness::Bool=false)
```

Comment:
special implementation, after discussion:

```julia
"""
    issubset(x::Interval, U::UnionSet{N, IT, IT}) where {N, IT<:Interval{N}}

Return whether an interval is contained in the union of two intervals.

### Input

- `x` -- interval
- `U` -- union of two intervals

### Output

`true` if `x ⊆ U` and `false` otherwise.

### Algorithm

Let ``U = a ∪ b `` where ``a`` and ``b`` are intervals. Then
the set difference ``y = x \ a`` can have three results:

1. If it is empty, then ``x ⊆ a`` hence ``x ⊆ a ∪ b``.

2. If the result is an interval, we check whether ``y ⊆ b``.

3. If the result is a union of intervals, it suffices to check if ``x ⊆ b``
   (since ``b` is an interval), see diagram below.

```
x      -------
a        ***
y      ++   ++
b    @@@@@@@@@@
```
"""
function issubset(x::Interval, U::UnionSet{N, IT, IT}) where {N, IT<:Interval{N}}
    a = U.X
    b = U.Y
    aux = difference(x, a)
    isempty(aux) && return true

    if isa(aux, Interval)
        aux2 = difference(aux, b)
        return isempty(aux2)
    else # union
        return x ⊆ b
    end
end
```

Reply:
the other option is to special case the n-ary union method:

```julia
function _issubset2(x::Interval, U::UnionSet{N, IT, IT}) where {N, IT<:Interval{N}}
    a = U.X
    b = U.Y
    if min(a) < min(b)
        # a is on the left of b
        min(x) < min(a) && return false
        return difference(x, a) ⊆ b
    else 
        # a is on the right of b
        min(x) < min(b) && return false
        return difference(x, b) ⊆ a
    end
end
```